Lib/fontTools/merge.py

```diff
@@ -566,6 +569,30 @@ def merge(self, m, tables):
 def mapLookups(self, lookupMap):
 	pass
 
+# Check if the given glyphs specified by gid are euqal or not.
+# Two glyphs are considered as equal iff:
+# 1. Outlines are equal.
+# 2. Advance width are equal.
+def isGlyphSame(fonts, gid_0, gid_1):
+	# Checks outline
+  assert '#' in gid_0 and '#' in gid_1
+  index_0 = int(gid_0.split('#')[-1])
+  index_1 = int(gid_1.split('#')[-1])
+  
+  assert fonts[index_0].has_key('glyf') and fonts[index_1].has_key('glyf')
+  glyfTable_0 = fonts[index_0]['glyf']
+  glyfTable_1 = fonts[index_1]['glyf']
+  data_0 = glyfTable_0[gid_0].compile(glyfTable_0)
```

Comment:
This doesn't work if the fonts have different hinting.  We still want to merge that case.
Worse, if the glyphs are composites and both refer to the same glyph indices in their original font, they do not necessarily look the same but this test will pass, and vice versa.